web/src/main/webapp/my-app/layout/controllers.js

```diff
@@ -47,11 +47,127 @@ define(['angular', 'jquery'], function(angular, $) {
     function($localStorage, $log, $sessionStorage,
       $scope, $rootScope, layoutService) {
       var vm = this;
+
+      /**
+       * Log whenever a widget is moved
+       * @param eventType {String} Kind of interaction that moved the widget
+       * @param fname {String} The moved widget's fname
+       * @param dropIndex {Number} Index where the widget landed
+       * @param startIndex {Number} (optional) Index before moving the widget
+       * @return {boolean}
```

Comment:
What are the semantics of the boolean returned? `true` means what? `false` means what?